app/controllers/internal_api/v1/pillars_controller.rb

```diff
@@ -129,5 +131,25 @@ def openstack_cloud_contents
       }
     }
   end
+
+  def kubelet_contents
+    reservations = {}
+    KubeletComputeResourcesReservation.all.each do |r|
+      reservations[r.component] = {
+        cpu: r.cpu,
+        memory: r.memory,
+        "ephemeral-storage" => r.ephemeral_storage
+      }
+    end
+
+    eviction_hard = Pillar.find_or_initialize_by(pillar: "kubelet:eviction-hard")
+
+    {
+      kubelet: {
+        "compute-resources" => reservations,
+        "eviction-hard"     => eviction_hard.value || ""
```

Comment:
for that to work it depends on https://github.com/kubic-project/salt/pull/533

Reply:
merged, thanks for the fix